user.js

```diff
@@ -195,57 +199,70 @@ module.exports = function user(options) {
     role: role,
     cmd:  'clean',
 
-    user:{required$:true,entity$:'sys/user'}, // sys/user entity
+    user:{required$:true,entity$:user_coll}, // sys/user entity
   }, cmd_clean )
 
 
+  // ### Lod user based on a query. Required by external seneca plugins such as seneca-auth
+  // Pattern: _**role**:user, **cmd**:load_user_
+  // Returns an entity based on a query
+  seneca.add(
+    {
+      role:role,
+      cmd:'load_user',
+      q:{object$:true,required$:true}
```

Comment:
using q in this way exposes implementation details
instead, merge all remaining properties into a query:

q = _.omit( args, ['role','get'] )